pkg/handler/branch.go

```diff
@@ -60,7 +60,13 @@ func (h *branchHandler) GetActiveBranches(w http.ResponseWriter, r *http.Request
 
 	var activeBranches []string
 	for _, branch := range repoBranches {
-		commit, _ := getCommit(owner, repo, branch.Commit.SHA)
+		commit, err := getCommit(owner, repo, branch.Commit.SHA)
+
+		if err != nil {
+			http.Error(w, "Commit not found", http.StatusInternalServerError)
```

Comment:
I think returning the same error from `getCommit` function would be more apt here. We can update "Commit not found" statement to **err.Error()** so we don't miss the original error message. What do you think?

Reply:
Yeah, that makes much more sense, thanks!
Sorry, I'm just struggling a bit with the syntax right now 😅

Reply:
No worries, it is perfectly fine. Thank you for the changes!